docs/en/modules/enduser-setup.md

```diff
@@ -1,13 +1,14 @@
 # enduser setup Module
 | Since  | Origin / Contributor  | Maintainer  | Source  |
 | :----- | :-------------------- | :---------- | :------ |
-| 2015-09-02 | [Robert Foss](https://github.com/robertfoss) | [Robert Foss](https://github.com/robertfoss) | [enduser_setup.c](../../../app/modules/enduser_setup.c)|
+| 2015-09-02 | [Robert Foss](https://github.com/robertfoss), [Jason Follas](https://github.com/jfollas) | [Robert Foss](https://github.com/robertfoss) | [enduser_setup.c](../../../app/modules/enduser_setup.c)|
```

Comment:
I wasn't trying to take credit or ownership away from @robertfoss , but just add accountability. I'm fine with not having anything modified in the docs.